src/test/java/org/mule/extension/file/integration/PostActionGroupTestCase.java

```diff
@@ -22,22 +22,22 @@ public class PostActionGroupTestCase extends AbstractMuleTestCase {
   @Test
   @Description("tests all the valid states of post action parameters")
   public void validAction() {
-    new PostActionGroup(true, null, null, true).validateSelf();
-    new PostActionGroup(true, null, null, false).validateSelf();
-    new PostActionGroup(false, "someDir", null, false).validateSelf();
-    new PostActionGroup(false, "someDir", "thisone.txt", false).validateSelf();
-    new PostActionGroup(false, null, "thisone.txt", false).validateSelf();
+    new PostActionGroup(true, null, null, true, false).validateSelf();
```

Comment:
You should add tests for the true value in the new parameter. 
(I have left a comment about the API change in the PostActionGroup class).

Reply:
Yes, I will add it. 
I just wait for build to be normal.